analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MemberOverrideCallsBaseMember.CSharp10.cs

```diff
@@ -46,14 +46,24 @@ record BaseWithOverride
         public override string ToString() => "Some custom ToString";
     }
 
-    record Derived: Base
+    record Derived : Base
     {
-        public override string ToString() => base.ToString(); // Compliant. Without the override the compiler generates a ToString instead of calling base.ToString
```

Comment:
The "failure" means that an issue isn't raised when it should? That was the behavior until now , I wouldn't change it in this PR. We now realize it, right?

I would keep things simple and consider this as an edge case. To avoid making the logic too complex.

And have two different tests:
- document the behavior for .NET Framework - add a comment to explain this
- have a different test for the .NET Core run of the UTs  to document the behavior there (we have conditional compilation in unit tests for this sort of scenarios)

like this the behavior will be documented.